scripts/fuzz_shell.js

```diff
@@ -178,9 +178,10 @@ function callFunc(func) {
     /* await */ func();
     return 0;
   } catch (e) {
-    // The exception must exist, and not behave oddly when we access a
-    // property on it. (VM bugs could cause errors here.)
-    e.a;
+    // The exception might be a JS null, but otherwise it must be valid to check
+    // if a property exists on it (VM bugs could cause errors here, specifically
+    // if a wasm exception is caught here, and it is not represented properly).
+    if (e !== null) e.a;
```

Comment:
Why `a`? All exceptions have the property `a`?

Reply:
It could be any property name - this just checks if any property can be read. Basically we test that it behaves like a proper JS object (return `undefined` for missing properties, return the value for existing properties, and either way not trapping).